azext_iot/iothub/command_map.py

```diff
@@ -33,6 +39,43 @@ def load_iothub_commands(self, _):
         cmd_group.show_command("show", "get_digital_twin")
         cmd_group.command("update", "patch_digital_twin")
 
+    with self.command_group("iot hub messaging-endpoint", command_type=iothub_message_endpoint_ops) as cmd_group:
+        cmd_group.show_command("show", "message_endpoint_show")
+        cmd_group.command("list", "message_endpoint_list")
+        cmd_group.command(
+            "delete", "message_endpoint_delete", transform=EndpointUpdateResultTransform(self.cli_ctx)
```

Comment:
Should we add confirmation to this command, since providing just the hub name (and no type/name parameters) will delete every endpoint on the hub?

Or perhaps we should require an `--all` switch (exclusive of type/name) that allows the deletion of all endpoints in one go?

Reply:
will just add a blanket confirmation since users can delete multiple endpoints by providing just the type too.